src/lib/routes/admin-api/instance-admin.ts

```diff
@@ -46,6 +60,10 @@ class InstanceAdminController extends Controller {
                 openApiService.validPath({
                     tags: ['Instance Admin'],
                     operationId: 'getInstanceAdminStats',
+                    summary:
+                        'An overview of usage of various features of Unleash',
```

Comment:
Maybe we can shorten it down a bit?

```suggestion
                        'Get feature usage overview',
```

Reply:
I feel like we've conflated the word feature here. Maybe "Get usage statistics" instead?

Reply:
Yeah, that's much better 😄

Reply:
Ended up using `Instance usage statistics`